### DISCUSSION TITLE
Clickable text that is not a button

### DISCUSSION THREAD

**Discussion by Ben-Lichtman** [kas-gui/kas#418] [Q&A]
What I want is to create some text which acts like a button but doesn't render as a button (maybe the text underlines when hovered?) is there some way to achieve this?

**Comment by Ben-Lichtman** [comment]
I think the correct term would be "link" here (ie. hyperlink)

**Comment by dhardy** [comment]
As a stand-alone widget? Yes, with a custom widget. You could just copy `Button` and remove the frame, and add some code for the underline (call `draw.text_effects` when you want the underline, `draw.text` when you don't).

Supporting hyperlinks embedded in text? No, that requires some significant overhaul of the text processing.
